proxmoxtf/resource/file.go

```diff
@@ -193,7 +193,39 @@ func File() *schema.Resource {
 		CreateContext: fileCreate,
 		ReadContext:   fileRead,
 		DeleteContext: fileDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: func(ctx context.Context, d *schema.ResourceData, i interface{}) ([]*schema.ResourceData, error) {
+				node, datastore, volumeID, err := fileParseImportID(d.Id())
+				if err != nil {
+					return nil, err
+				}
+
+				d.SetId(volumeID)
+
+				err = d.Set(mkResourceVirtualEnvironmentFileNodeName, node)
+				if err != nil {
+					return nil, fmt.Errorf("failed setting state during import: %w", err)
+				}
+
+				err = d.Set(mkResourceVirtualEnvironmentFileDatastoreID, datastore)
+				if err != nil {
+					return nil, fmt.Errorf("failed setting state during import: %w", err)
+				}
+
+				return []*schema.ResourceData{d}, nil
+			},
+		},
+	}
+}
+
+func fileParseImportID(id string) (string, string, string, error) {
```

Comment:
I think this function should have unit tests to capture the logic of splitting and joining ID parts. Might be worth moving it to `utils.go` as well.

Reply:
I think we should also add an "Import" section to documentation for all resources that support import, and provide and example of the import command with import ID, for example: [aws instance import](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#import)

Reply:
Not sure why it's worth moving to `utils.go`, it's only used here

Reply:
done